Servers/Administration/OJS.Servers.Administration/Controllers/ProblemsController.cs

```diff
@@ -430,16 +430,16 @@ await this.contestsValidationHelper
             IsReadOnly = true,
         });
 
-        if (!contest.IsOnlineExam)
+        formControls.Add(new FormControlViewModel
         {
-            formControls.Add(new FormControlViewModel
-            {
-                Name = AdditionalFormFields.ProblemGroupType.ToString(),
-                Options = EnumUtils.GetValuesFrom<ProblemGroupType>().Cast<object>(),
-                Type = typeof(ProblemGroupType),
-                Value = entity.ProblemGroup?.Type ?? default(ProblemGroupType),
-            });
+            Name = AdditionalFormFields.ProblemGroupType.ToString(),
+            Options = EnumUtils.GetValuesFrom<ProblemGroupType>().Cast<object>(),
+            Type = typeof(ProblemGroupType),
+            Value = entity.ProblemGroup != null ? entity.ProblemGroup.Type ?? default(ProblemGroupType) : default,
+        });
 
+        if (!contest.IsOnlineExam)
+        {
```

Comment:
Is this the same logic in Legacy judge? I believe ProblemGroupType cannot be edited there. You should use the default value, instead of allowing the admin to edit it.

Reply:
In Legacy problem group type CAN be edited for contest which is not Online, but in Online you cannot. What is the same? I see that in the PR you allow editing of problem group type for Online contest, or I'm missing something?

Reply:
Fixed the logic. Now in an online contests for problems, problemGroupType is visible, and for others- problemGroupType.